src/main/g8/project/plugins.sbt

```diff
@@ -1,3 +1,2 @@
-addSbtPlugin("org.scalatra.scalate" % "sbt-scalate-precompiler" % "1.8.0.1")
-
+addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.3.2")
 addSbtPlugin("org.scalatra.sbt" % "scalatra-sbt" % "0.5.1")
```

Comment:
In addition, should you also update the version of scalatra-sbt? (At least 0.6.0)

Reply:
Indeed!